src/pages/news/news-page/news.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { watchNewsLoad } from '../../../redux/news/news.actions';
+import { useStyles } from './news.style';
+import NewsPageItem from '../news-list';
+import { LANGUAGE } from '../../../configs';
+
+const NewsPage = ({ watchNewsLoad, list }) => {
+  useEffect(() => {
+    watchNewsLoad();
+    window.scrollTo(0, 0);
+  }, [watchNewsLoad]);
+
+  const newsHeader = ['Новини', 'News'];
+  const styles = useStyles();
```

Comment:
Please, move styles inside `NewsPageItem` component

Reply:
why? from read me: "all styles should be located inside a separate file {modelName}-styles.js in the component folder."